zerver/lib/send_email.py

```diff
@@ -14,8 +14,13 @@
 import os
 from typing import Any, Dict, Iterable, List, Mapping, Optional, Text
 
+
+from zerver.lib.logging_util import create_logger
```

Comment:
Do you need this import in your diff?

Reply:
I am not using that import . I don't need it.

Reply:
Can you remove it from your commit, please?

Reply:
I removed it .